Library/Homebrew/brew.sh

```diff
@@ -454,8 +454,23 @@ setup_git() {
   fi
 }
 
+setup_gh() {
```

Comment:
Do we need to handle non-brewed `gh` just now (or ever)?

Reply:
IMO no to both -- will replace with `ensure_executable!`

Reply:
Great!